File: pbwebcert/pbwebcert-jar/src/main/java/it/csi/pbandi/pbwebcert/integration/request/InviaReportRequest.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebcert.integration.request;

import it.csi.pbandi.pbwebcert.dto.certificazione.PropostaCertificazioneDTO;

public class InviaReportRequest {
	 private PropostaCertificazioneDTO propostaCertificazione;
	public PropostaCertificazioneDTO getPropostaCertificazione() {
		return propostaCertificazione;
	}
	public void setPropostaCertificazione(PropostaCertificazioneDTO propostaCertificazione) {
		this.propostaCertificazione = propostaCertificazione;
	}
	
}

File: pbwebcertwcl/src/app/gestione-proposte/commons/requests/invia-report-request.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { PropostaCertificazioneDTO } from "src/app/shared/commons/dto/proposta-certificazione-dto";

export class InviaReportRequest {
    constructor(
        public propostaCertificazione: PropostaCertificazioneDTO
    ) { }
}